sample-android/src/test/java/com/github/takahirom/roborazzi/sample/ManualTest.kt

```diff
@@ -177,6 +178,9 @@ class ManualTest {
       roborazziComposeOptions = RoborazziComposeOptions {
         // We have several options to configure the test environment.
         fontScale(2f)
+        /* The default value is false, but we can set it to true,
+        if you want to use the logic for Preview in composable functions. */
+        localInspectionMode(false)
```

Comment:
I don't think we need to apply localInspectionMode explicitly. How about making it a comment?

```suggestion
        /*
        We don't specify `localInspectionMode` by default. The default value for `localInspectionMode` in Compose is `false`. This is to maintain higher fidelity in tests.
        If you encounter issues integrating the library, you can set `localInspectionMode` to `true`.
        localInspectionMode(true)
        */

Reply:
@takahirom 
OK, I have incorporated your feedback into the comment.
https://github.com/takahirom/roborazzi/pull/588/commits/4f6007ad6296d48abf4c264b7d6ccc097176dc7c